mutect2.wdl

```diff
@@ -34,12 +44,34 @@ workflow mutect2 {
     }]
```

Comment:
Since _gatk_ is a parameter, it probably needs to be removed from the list of dependencies

Reply:
I removed it from dependency and added meta for gatk in wdl, as well as other parameter meta.